src/seedu/addressbook/data/person/Unit.java

```diff
@@ -0,0 +1,13 @@
+package seedu.addressbook.data.person;
+
+public class Unit {
+    private final String value;
+    
+    public Unit(String unit){
```

Comment:
need a space between the braces and the text; what do coding standards say?  ;-)